src/commands/utils_command/generate_keypair_subcommand/mod.rs

```diff
@@ -58,7 +39,7 @@ impl CliGenerateKeypair {
                 println!(
                     "Master Seed Phrase: {}\nSeed Phrase HD Path: {}\nImplicit Account ID: {}\nPublic Key: {}\nSECRET KEYPAIR: {}",
                     key_pair_properties.master_seed_phrase,
-                    bip32path_to_string(&key_pair_properties.seed_phrase_hd_path),
+                    crate::common::bip32path_to_string(&key_pair_properties.seed_phrase_hd_path),
```

Comment:
HdPath now supports Display and ToString, so we can just drop this helper, and here you just let `key_pair_properties.seed_phrase_hd_path` to be formatted for display.